src/AboutDialog.cpp

```diff
@@ -83,774 +85,13 @@ static const auto ProgramName =
    //XO("Audacity");
    Verbatim("Audacity");
 
-void AboutDialog::CreateCreditsList()
```

Comment:
Large number of moved lines makes this diff hard to understand

Reply:
Sorry I must return to this commit later today.

Reply:
A huge text blob in the middle of the layout made it very hard to read and debug. Probably, I should move the license blob to a different file, so the diff will be shorter